CircleciScripts/prepare_xcframework_archives.py

```diff
@@ -0,0 +1,80 @@
+import os
+import sys
+import shutil
+import re
+
+from framework_list import xcframeworks
+from functions import log, run_command
+
+def create_archives(xcframework_path, archive_path, version):
+    os.makedirs(archive_path, exist_ok=True)
+    for framework in xcframeworks:
+
+        finalZipName = f"{framework}-{version}"
+        log(f"Creating zip file for {finalZipName}")
+
+        xcframework_sdk_path = f"{xcframework_path}/{framework}.xcframework"
+        temp_path = f"{xcframework_path}/{framework}/{framework}.xcframework"
```

Comment:
For clarity, it would be good to build up the path, since you remove the parent dir (xcframework_path}/framework) later.

Reply:
Fixed